app/services/create_nomis_visit.rb

```diff
@@ -71,7 +72,7 @@ def booking_params
       override_visitor_restrictions: false,
       override_vo_balance: false,
       override_slot_capacity: false,
-      client_unique_ref: visit.id
+      client_unique_ref: visit.id, comment: creator.email
```

Comment:
new line

Reply:
creator.email needs to be passed to NOMIS via pvb_user_id not comment

Reply:
@TobyRet where do you see that pvb_user_id field in API docs?
I think it needs to be passed as through the `X-PVB-User-id` header.
https://github.com/ministryofjustice/nomis-api/blob/dev/src/main/java/syscon/nomisapi/visit/OffenderVisits.java#L628